### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DominiqueMakowski**

### CODE REVIEW

File: R/correlation.R [hunk 1]
```diff
@@ -44,7 +41,7 @@ correlation <- function(data, data2 = NULL, ci = "default", method = "pearson",
   if (bayesian == FALSE) {
     if (ci == "default") ci <- 0.95
   } else {
-    if (ci == "default") ci <- 0.9
+    if (ci == "default") ci <- 0.89
   }
 
   if (inherits(data, "grouped_df")) {
```
Comment (DominiqueMakowski) [inline R/correlation.R]: @strengejacke Below here are the remaining instances of dplyr, related to the grouped_df thing. Not sure how to replace them... Any ideas?

Comment (strengejacke) [reply R/correlation.R]: We have done this already in one or two other packages. I'll look into this, we can also leave this to a further update. not sure when you plan to submit _correlation_? Next week, I'm on a congress and give some workshops, so time is rare the next days...

Comment (DominiqueMakowski) [reply R/correlation.R]: > We have done this already in one or two other packages

I think this one's a bit trickier, cause there is some ungroup involved

> not sure when you plan to submit correlation

No deadline, but I see no reason to delay either. Can always try to get rid of dplyr for v2, and put it to suggests in the meantime

Comment (strengejacke) [reply R/correlation.R]: If I understand the code, you just need the group vars. Ungrouping is not necessary as you don't call dplyr-functions in the further processing anyway.

**Pull request merged**